libretroshare/src/retroshare/rsgossipdiscovery.h

```diff
@@ -64,6 +64,8 @@ struct RsGossipDiscoveryFriendInviteReceivedEvent : RsEvent
 class RsGossipDiscovery
 {
 public:
+    virtual ~RsGossipDiscovery() {}
```

Comment:
`= default;` should be used instead if `{}` because the compiler can do more optimizations